Tests/test_imagefont.py

```diff
@@ -856,6 +856,19 @@ def test_bitmap_font_stroke(layout_engine):
     assert_image_similar_tofile(im, target, 0.03)
 
 
+@pytest.mark.parametrize("embedded_color", (False, True))
+def test_bitmap_blend(layout_engine, embedded_color):
+    font = ImageFont.truetype(
+        "Tests/fonts/EBDTTestFont.ttf", size=64, layout_engine=layout_engine
+    )
+
+    im = Image.new("RGBA", (128, 96), "white")
+    d = ImageDraw.Draw(im)
+    d.text((16, 16), "AA", font=font, embedded_color=embedded_color, fill="#8E2F52")
```

Comment:
```suggestion
    d.text((16, 16), "AA", font=font, fill="#8E2F52", embedded_color=embedded_color)
```
Updates the argument order to match
https://github.com/python-pillow/Pillow/blob/316f39702c16ed97a95b971cb4aad8a87a26f9df/Tests/test_imagefont.py#L866